progress/writer.go

```diff
@@ -4,6 +4,10 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
Lets not add a new vendor to get console size.  Just use `github.com/containerd/console` that is already vendored and used in the client and elsewhere.

Reply:
Also it maybe better to pass in width/height to the writer on initialization than it is to get it inside the package.

Reply:
I didn't notice we've already got a console package, let me try to modify it as you suggested :)

Reply:
@crosbymichael Do you really think it's better to pass in width/height? I am thinking that maybe we should embed the "get console size" logic inside progress package, because I don't think the caller should care about win size things.  Do you have any concerns about this?

Reply:
@WeiZhang555 ok, i didn't notice the lines were already part of the writer.  SGTM